client/src/main/java/io/split/engine/matchers/EqualToSemverMatcher.java

```diff
@@ -0,0 +1,54 @@
+package io.split.engine.matchers;
+
+import io.split.engine.evaluator.EvaluationContext;
+
+import java.util.Map;
+
+public class EqualToSemverMatcher implements Matcher {
+
+    private final Semver _semVer;
+
+    public EqualToSemverMatcher(String semVer) {
+        _semVer = Semver.build(semVer);
+    }
+
+    @Override
+    public boolean match(Object matchValue, String bucketingKey, Map<String, Object> attributes, EvaluationContext evaluationContext) {
+        if (matchValue == null) {
```

Comment:
make the check if `_semVer == null here` please.

if `_semVer` is null you don't need to build the `matchSemver` instance.

Reply:
done